flake.nix

```
@@ -101,7 +105,7 @@
                 buildInputs = [
                   pkgs.go
                   pkgs.perl
-                ];
+                ] ++ (pkgs.lib.optionals pkgs.stdenv.isDarwin apple_sdk);
```

nitpick: Do we need to check `isDarwin` and the system? Also, if it's just one package then we can use the `lib.optional` function instead.

```suggestion
                ]
                # On intel macs, the default SDK is still 10.12 and Holochain won't build against that because we're
                # using a newer Go version. So override with the newest SDK available for x86_64-darwin.
                ++ (pkgs.lib.optional (if system == "x86_64-darwin") pkgs.apple-sdk_10_15);
```
This really is just a style thing though so I leave the decision up to you :slightly_smiling_face:

---

:+1:  It's a little more precise, though it makes no difference to the aarch64 systems.

---

We don't really no, but since this change is being backported, I'd rather not just fix this on 0.3. I'll put a PR to tidy this up on `main` and leave main-0.3 and main-0.4 as they are